doc/whats_new/v1.1.rst

```diff
@@ -258,6 +258,10 @@ Changelog
   Setting a transformer to "passthrough" will pass the features unchanged.
   :pr:`20860` by :user:`Shubhraneel Pal <shubhraneel>`.
 
+- |Fix| :class: `pipeline.Pipeline` now does not `._validate_steps()` in
```

Comment:
Let's not mention private API in the release notes as they are subject to change without warning in future versions:

```suggestion
- |Fix| :class: `pipeline.Pipeline` now does not validate hyper-parameters in
```